KerbalVR_Mod/KerbalVR/InteractionCommon/VRFlightStick.cs

```diff
@@ -152,16 +152,18 @@ private void GetInput(FlightCtrlState st)
 			}
 		}
 
-		private void OnGrab(Hand hand)
+		private void OnGrab(Hand hand, SteamVR_Input_Sources source)
```

Comment:
There is Hand.handType for this (maybe needs a better name)

Reply:
Fixed in 767ad21.